lib/elixir_analyzer/exercise_test/common_checks.ex

```diff
@@ -47,6 +48,9 @@ defmodule ElixirAnalyzer.ExerciseTest.CommonChecks do
       ModulePascalCase.run(code_ast),
       CompilerWarnings.run(code_path, code_ast),
       BooleanFunctions.run(code_ast),
+      Indentation.run(code_ast, code_string),
+      FunctionAnnotationOrder.run(code_ast),
+      Comments.run(code_ast, code_string),
```

Comment:
I think you ended up adding extra `Indentation` and `Comments` functions in the merge, that's why there is a test failing. Those two are repeated below.